### DISCUSSION TITLE
Install on top of raspbian

### DISCUSSION THREAD

**Discussion by josefhernandez** [Ysurac/openmptcprouter#3118] [General]
Hi,

I've a raspbian portable device (battery, 2 x 4G modem) and other sw installed on a raspberry pi 4
Can I install openmtptcp directly on raspbian without installing the provided image?
I need a desktop environment with audio support and chromium for my project

Best regards

**Comment by Ysurac** [comment]
OpenMPTCProuter is an OS, based on OpenWRT. It can't run under Raspbian or you need to create a VM (I don't think it's a good idea on a RPI).
